Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Scenes/MaterialGallery.unity

```diff
@@ -115,7 +115,7 @@ NavMeshSettings:
   m_NavMeshData: {fileID: 0}
 --- !u!4 &7969012 stripped
 Transform:
-  m_PrefabParentObject: {fileID: 4085006294760784, guid: 6e6521182b532d245b0d80293b332399,
```

Comment:
Not sure what the issue is here, but I'm missing many prefab references in basically all the scenes. It doesn't like any guids have changed though.

![image](https://user-images.githubusercontent.com/3580640/43491665-47f9f600-94da-11e8-9ff2-3e46c6691e3d.png)

Reply:
I just restarted Unity and I'm seeing this too, which is very concerning... let me se what is going on.

Reply:
I was just about to ask if anyone has seen this, the guids look the same. And, I haven't touched those prefabs... has anyone seen something like this before? I'm going to keep digging.

Reply:
Yeah, I'm not seeing any messed up metas on my end either. I tried regenerating the Library folder with no success.

Reply:
I need to step out for today, but I did notice it appears Unity changed the serialized names of 3 prefab tracking variables (looks like an innocent name change) in the scene assets. Reverting the scene restores the prefab linkage, but I would like to see if I can find a proper fix.

Reply:
Found a real fix, I want to test one thing then I'll push the changes. This was the problem for those interested: 

I needed to replace 4 text strings in the scene: 
"serializedVersion: 6" => "serializedVersion: 5" 
"m_CorrespondingSourceObject" => "m_PrefabParentObject" 
"m_IsPrefabAsset" => "m_IsPrefabParent" 
"m_SourcePrefab" => "m_ParentPrefab"


Source: https://answers.unity.com/questions/1520106/what-does-assertion-failed-on-expression-prefabisp.html